### PR TITLE
[docs] add document for running BR tests locally

### PR DESCRIPTION
**Pull request by wgtdkp**

### CODE REVIEW

File: tests/scripts/thread-cert/border_router/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+## Run Border Router (BR) tests locally
+
+BR tests run in isolated Docker network and containers, so new OTBR Docker image needs to be created before running the tests:
+
+```shell
+# Clear current OpenThread directory (remember to add new source files).
+git clean -xfd
+
+# Rebuild the OTBR Docker image if OTBR source code is updated.
+./script/test build_otbr_docker
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/border_router/README.md:12]: nit: We can use `TREL=1 ./script/test build_otbr_docker` to enable TREL.

File: tests/scripts/thread-cert/border_router/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+./script/test get_thread_wireshark
+
+# Build simulated OpenThread firmware.
+VIRTUAL_TIME=0 ./script/test build
+
+# Run the BR tests locally.
+VERBOSE=1 PACKET_VERIFICATION=1 VIRTUAL_TIME=0 ./script/test cert_suite ./tests/scripts/thread-cert/border_router/test_advertising_proxy.py
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/border_router/README.md:20]:
```suggestion
sudo VERBOSE=1 PACKET_VERIFICATION=1 VIRTUAL_TIME=0 ./script/test cert_suite ./tests/scripts/thread-cert/border_router/test_advertising_proxy.py
```

File: tests/scripts/thread-cert/border_router/README.md (hunk 1 above)
Comment (simonlingoogle) [reply tests/scripts/thread-cert/border_router/README.md:12]: Add command on how to use `LOCAL_OTBR_DIR`?

File: tests/scripts/thread-cert/border_router/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+BR tests run in isolated Docker network and containers, so new OTBR Docker image needs to be created before running the tests:
+
+```shell
+# Clear current OpenThread directory (remember to add new source files).
+git clean -xfd
+
+# Rebuild the OTBR Docker image if OTBR source code is updated.
+./script/test build_otbr_docker
+
+./script/test get_thread_wireshark
```
Comment (superwhd) [inline tests/scripts/thread-cert/border_router/README.md:14]: This step is only needed when /tmp is cleaned. This script will delete and re-download the library if you re-run it, which is unnecessary.

File: tests/scripts/thread-cert/border_router/README.md (hunk 2 above)
Comment (superwhd) [inline tests/scripts/thread-cert/border_router/README.md:20]: Replace `test_advertising_proxy.py` with a variable to keep generality?

**Review by superwhd — APPROVED** [review]
LGTM 👍

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1:

File: tests/scripts/thread-cert/border_router/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# OpenThread Border Router Tests
+
+## Run Border Router (BR) tests locally
+
+BR tests run in isolated Docker network and containers, so new OTBR Docker image needs to be created before running the tests:
```
Comment (sarah-iot) [inline tests/scripts/thread-cert/border_router/README.md:5]: LGTM - might want to say: 
BR tests run in isolated Docker network and containers, so a new OTBR Docker image needs to be created before running these tests:

**Review by sarah-iot — APPROVED** [review]
LGTM

File: tests/scripts/thread-cert/border_router/README.md (hunk 4 above)
Comment (jwhui) [reply tests/scripts/thread-cert/border_router/README.md:5]:
```suggestion
BR tests run in isolated Docker network and containers, so a new OTBR Docker image needs to be created before running these tests:
```

**Review by jwhui — APPROVED** [review]

**Pull request merged**